client/common/cmdline.c

```diff
@@ -144,11 +148,11 @@ static char* name_from_path(const char* path)
 	const char* name = "NULL";
 	if (path)
 	{
-		if (_strnicmp(path, "%", 2) == 0)
+		if (option_equlas("%", path))
```

Comment:
Typo: `option_equlas` -> `option_equals`

Reply:
ok, fixed. had every possible turner of characters in here :/